README.md

```diff
@@ -109,6 +112,13 @@ The following options are available:
      --marketplace-address  Address of deployed Marketplace contract.
      --validator            Enables validator, requires an Ethereum node [=false].
      --validator-max-slots  Maximum number of slots that the validator monitors [=1000].
```

Comment:
If we decide to allow `--validator-max-slots=0` means validate all slots, then we should add that to the help